config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
We really need to log the machine type in the `cc-env` output (currently it only shows the path to the hypervisor). Since that is an oversight, I don't think that is necessary for this PR (but it would be nice if you want to add it :). If not, let me know so I can raise an issue.